modules/core/include/opencv2/core/mat.hpp

```diff
@@ -1717,6 +1720,12 @@ class CV_EXPORTS Mat
      */
     size_t total() const;
 
+    /** @brief Returns the total number of array elements.
+
+     The method returns the number of elements within a certain sub-array slice with startDim <= dim < endDim
+     */
+    size_t total(int startDim, int endDim=INT_MAX) const;
```

Comment:
`sliceSize` is no better than `total`, `sliceTotal` does not sound right either. The method is small, but may be useful when you operate on tensors, in dnn this method is used several times